doc/api/process.md

```diff
@@ -129,7 +129,8 @@ unforeseen and unpredictable issues.
 
 Exceptions thrown from within the event handler will not be caught. Instead the
 process will exit with a non zero exit code and the stack trace will be printed.
-This is to avoid infinite recursion.
+This is to avoid infinite recursion. There may be other situations where the
```

Comment:
Well, I think the point is that it is not guaranteed to run for all application 'crashes', where 'crash' is pretty loosely defined. I can send `$ kill -9 NODE_PID` from the command line to my Node.js process, and the `uncaughtException` event is never emitted. Granted, there is no exception in this case, but I think that's what the docs are intended to mean. A crash does not necessarily mean `uncaughtException` is emitted.

How about, "There may be other situations where the application could crash and not trigger this event. For example, receiving a `SIGKILL` will cause the application to exit abruptly, but an `uncaughtException` event will not be emitted."?